### DISCUSSION TITLE
Using ESP-S host mode

### DISCUSSION THREAD

**Discussion by khayamgondal** [luc-github/ESP3D-TFT#60] [Q&A] [answered]
Can I use an ESP S and directly connect it to the printer via USB C port since S series supports USB host mode?

**Comment by luc-github** [comment, marked as answer]
you need to define the board and pins and use a dual core to get correct performance so only S3

**Comment by khayamgondal** [comment]
Thanks, are there any instructions on how to do that with an ESP-S3? I thought one of the purpose of your project was to implement that using IDF 5.1

**Comment by luc-github** [comment]
this project is use esp32 to handle a TFT and already support this feature for TFT based on ESP32-S3 https://github.com/luc-github/ESP3D-TFT/tree/main/hardware/ESP32S3_HMI43V3 some board without TFT may work but I  did not tested : https://github.com/luc-github/ESP3D-TFT/tree/main/hardware/ESP32S3_FREENOVE_1_1 because not in my short term target
